sspl_test/alerts/self_hw/self_hw_utilities.py

```diff
@@ -37,24 +38,9 @@ def is_virtual():
     return retVal
 
 def get_node_id():
-        GRAINS_GET_NODE_CMD = "sudo salt-call grains.get id --output=newline_values_only"
-        MINION_GET_NODE_CMD = "cat /etc/salt/minion_id"
         node_key = 'srvnode-1'
         try:
-            subout = subprocess.Popen(GRAINS_GET_NODE_CMD, shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-            result = subout.stdout.readlines()
-            if result == [] or result == "":
-                print(f"Command '{GRAINS_GET_NODE_CMD}' failed to fetch grain id or hostname.")
-                subout = subprocess.Popen(MINION_GET_NODE_CMD, shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-                result = subout.stdout.readlines()
-                if result == [] or result == "":
-                    print(f"Command '{MINION_GET_NODE_CMD}' failed to fetch minion id or hostname.")
-                    print("Using node_id ('srvnode-1') as we are not able to fetch it from hostname command.")
-                    node_key = 'srvnode-1'
-                else:
-                    node_key = result[0].decode().rstrip('\n').lower()
-            else:
-                node_key = result[0].decode().rstrip('\n')
+            node_key = Conf.get(GLOBAL_CONF, NODE_KEY)
```

Comment:
collection should happen in node_id. I know its existing code, lets change that too now

Reply:
done.